app/src/main/java/io/zenandroid/onlinego/ui/views/BoardView.kt

```diff
@@ -488,11 +490,29 @@ class BoardView : View {
         private var texture: Bitmap? = null
 
         @Synchronized
-        fun preloadResources(resources: Resources) {
+        fun preloadResources(resources: Resources, forceTextureRedraw: Boolean = false ) {
```

Comment:
I really don't think this needs to be that complicated... I will rewrite this once the rest is merged to keep it simple.

Reply:
now it's much simpler :)